common/translations/lang/en.json

```diff
@@ -81,9 +81,11 @@
     "x_Json": "JSON File (unencrypted) ",
     "x_JsonDesc": "This is the unencrypted, JSON format of your private key. This means you do not need the password but anyone who finds your JSON can access your wallet & Ether without the password. ",
     "x_Keystore": "Keystore File (UTC / JSON · Recommended · Encrypted) ",
-    "x_Keystore2": "Keystore File (UTC / JSON) ",
+    "x_Keystore2": "Keystore File ",
```

Comment:
Is this change OK? Or should I make `x_Keystore3`?

Reply:
@tayvano Feel free to chime in with your thoughts.

My understanding is we'll do a once-over for all of the copy, and can address issues at that point.